src/Raven.Server/Documents/AI/Embeddings/EmbeddingValue.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using Sparrow.Server.Utils;
+
+namespace Raven.Server.Documents.AI.Embeddings;
+
+public class EmbeddingValue : IEmbeddingValue
```

Comment:
What does this gives us that `Memory<byte>.Slice()` does not?

Reply:
ReadOnlySpan<byte> from ReadOnlyMemory<float> (and slicing when performed quantization)